fMRISurface/GenericfMRISurfaceProcessingPipeline.sh

```diff
@@ -1,24 +1,24 @@
-#!/bin/bash 
+#!/bin/bash
 set -e
 
 # Requirements for this script
-#  installed versions of: FSL (version 5.0.6), FreeSurfer (version 5.3.0-HCP) , gradunwarp (HCP version 1.0.2) 
+#  installed versions of: FSL (version 5.0.6), FreeSurfer (version 5.3.0-HCP) , gradunwarp (HCP version 1.0.2)
```

Comment:
@coalsont I've always found the wording `# Requirements for this script` confusing in the context of listing specific versions of the tools.  e.g., It could be interpreted as FSL 5.0.6 is the _only_ supported version.  I think we should either change to `# Minimum requirements for this script`, or simply list the tools without specific version numbers (or perhaps even both).

Reply:
Minimum tool versions might be useful, though we give such requirements for the pipelines as a whole, and updating them in each script would be tedious and error-prone.  Removing the version numbers from the scripts certainly has some appeal as the laziest way forward.

Reply:
Captured for us to do as Item 8 of #146.

Reply:
Ok. I guess not an issue here.